src/libcollections/btree/node.rs

```diff
@@ -669,7 +601,10 @@ impl<'a, K, V> NodeRef<marker::Mut<'a>, K, V, marker::LeafOrInternal> {
                 ForceResult::Leaf(_) => None,
                 ForceResult::Internal(internal) => {
                     let edge = ptr::read(internal.as_internal().edges.get_unchecked(idx + 1));
-                    let mut new_root = Root { node: edge, height: internal.height - 1 };
+                    let mut new_root = Root {
+                        node: edge,
+                        height: internal.height - 1,
+                    };
```

Comment:
This looks fine (in fact, I like the expanded look a bit better), but it surprises me given that `rustfmt` kept some longer and more cumbersome `struct` initializers as a single line farther up.